core/domain/exp_domain.py

```diff
@@ -865,7 +861,7 @@ def __init__(self, hint_text):
         Args:
             hint_text: str. The hint text.
         """
-        self.hint_text = html_cleaner.clean(hint_text)
+        self.hint_text = hint_text
```

Comment:
It's not really just text any more, though...

I wonder, should we get rid of this class together and treat it similar to what we did for Content -- i.e. just use the SubtitledHtml class to replace it?

Reply:
changed to hint_content